modules/ROOT/pages/changelog.adoc

```diff
@@ -4,9 +4,59 @@
 
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
-== 7.0.1 - 2024-05-08
+== 7.1 - 2024-05-08
 
-TBA
+### Added
+- New `math-equation` icon.
+// #TINY-10804
+
+### Improved
+- Included itemprop, itemscope, itemtype as valid HTML5 attributes in the core schema.
+// #TINY-9932
+- Notification accessibility improvements: added tooltips, keyboard navigation and shortcut to focus on notifications.
+// #TINY-6925
+- Removed `aria-pressed` from the `More` button in sliding toolbar mode and replaced it with `aria-expanded`.
+// #TINY-10795
+- The editor UI now renders correctly in Windows High Contrast Mode.
+// #TINY-10781
+
+### Fixed
+- Backspacing in certain html setups resulted in data moving around unexpectedly.
+// #TINY-10590
+- Dialog title markup changed to use an `h1` element instead of `div`.
+// #TINY-10800
+- Dialog title was not announced in macOS VoiceOver, dialogs now use `aria-label` instead of `aria-labelledby` on macOS.
+// #TINY-10808
+- Theme loader did not respect the suffix when it was loading skin CSS files.
+// #TINY-10602
+- Custom block elements with colon characters would throw errors.
+// #TINY-10813
+- Tab navigation in views didn't work.
+// #TINY-10780
+- Video and audio elements could not be played on Safari.
+// #TINY-10774
+- `ToggleToolbarDrawer` command did not toggle the toolbar in `sliding` mode when `+{skipFocus: true}+` parameter was passed.
+// #TINY-10726
+- The buttons in the custom view header were clipped when overflowing.
+// #TINY-10741
+- In the custom view, the scrollbar of the container were not visible if its height was greater than the editor.
+// #TINY-10741
+- Fixed accessibility issue by removing duplicate `role="menu"` attribute from color swatches.
+// #TINY-10806
+- Fullscreen mode now prevents focus from leaving the editor.
+// #TINY-10597
+- Open link context menu action did not work with selection surrounding a link.
+// #TINY-10391
+- Styles were not retained when toggling a list on and off.
+// #TINY-10837
+- Caret and placeholder text were invisible in Windows High Contrast Mode.
+// #TINY-9811
+- Firefox not announcing the iframe title when `iframe_aria_text` was set.
+// #TINY-10718
+- Notification width was not constrained to the width of the editor.
+// #TINY-10886
+- Open link context menu action was not enabled for links on images.
+// #TINY-10391
```

Comment:
```suggestion
### Added

- New `math-equation` icon.
// #TINY-10804

### Improved

- Included `itemprop`, `itemscope`, and `itemtype` as valid HTML5 attributes in the core schema.
// #TINY-9932
- Improved accessibility for notifications: added tooltips, keyboard navigation, and a shortcut to focus on notifications.
// #TINY-6925
- Removed `aria-pressed` from the `More` button in sliding toolbar mode and replaced it with `aria-expanded`.
// #TINY-10795
- The editor UI now renders correctly in Windows High Contrast Mode.
// #TINY-10781

### Fixed

- Backspacing in certain HTML setups caused data to move around unexpectedly.
// #TINY-10590
- Dialog title markup was changed to use an `h1` element instead of a `div`.
// #TINY-10800
- The dialog title was not announced in macOS VoiceOver; dialogs now use `aria-label` instead of `aria-labelledby` on macOS.
// #TINY-10808
- The theme loader did not respect the suffix when loading skin CSS files.
// #TINY-10602
- Custom block elements with colon characters would throw errors.
// #TINY-10813
- Tab navigation in views didn't work.
// #TINY-10780
- Video and audio elements couldn't be played on Safari.
// #TINY-10774
- The `ToggleToolbarDrawer` command did not toggle the toolbar in `sliding` mode when the `skipFocus: true` parameter was passed.
// #TINY-10726
- The buttons in the custom view header were clipped when overflowing.
// #TINY-10741
- In the custom view, the scrollbar of the container was not visible if its height was greater than the editor.
// #TINY-10741
- Fixed an accessibility issue by removing the duplicate `role="menu"` attribute from color swatches.
// #TINY-10806
- Fullscreen mode now prevents focus from leaving the editor.
// #TINY-10597
- The "Open Link" context menu action did not work when the selection surrounded a link.
// #TINY-10391
- Styles were not retained when toggling a list on and off.
// #TINY-10837
- Caret and placeholder text were invisible in Windows High Contrast Mode.
// #TINY-9811
- Firefox did not announce the iframe title when `iframe_aria_text` was set.
// #TINY-10718
- Notification width was not constrained to the width of the editor.
// #TINY-10886
- The "Open Link" context menu action was not enabled for links on images.
// #TINY-10391
```